test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -561,3 +561,22 @@ func ResourceInfoProvidingAtMost(resources []nrtv1alpha2.ResourceInfo, resName s
 	}
 	return true
 }
+
+func EqualNRTListsItems(nrtListA, nrtListB nrtv1alpha2.NodeResourceTopologyList) (bool, error) {
```

Comment:
optional followup: can this be placed or reimplemented in terms of code in `internal/noderesourcetopology/equality.go`?

Reply:
completely agree. This is non-urgent and can wait a followup PR later on.